addons/web_editor/static/src/js/common/utils.js

```diff
@@ -460,6 +460,16 @@ function _shouldEditableMediaBeEditable(mediaEl) {
         && nonEditableAncestorRootEl.parentElement.isContentEditable;
 }
 
+/**
+ * Removes all zero-width spaces from a string.
+ *
+ * @param {string} str
+ * @returns {string}
+ */
+function _removeZWS(str) {
+    return str.replaceAll("\u200B", "");
+}
```

Comment:
I would go with something more explicit for link element labels. That would actually make it less weird this is not part of the editor lib itself (which has a `cleanZWS` function). That low-level operation is actually done in the editor so we could ask "why not using the util", e.g. for `sel.toString().replace(/\u200B/g, '')`

So maybe something like this would be better and more "editor panel related":
```javascript
function getLinkLabel(linkEl) {
    return linkEl.innerText.trim().replaceAll('\u200B', '');
}
```
It would also consolidate the use of `trim`, always.